example/Demo_make_simlibs.ipynb

```diff
@@ -42,18 +42,21 @@
     "\n",
     "### 1. Download the desired file<a name=\"download\"></a>\n",
     "\n",
-    "In this example, we will download the `baseline_v1.4_10yrs.db` file from [here](https://lsst-web.ncsa.illinois.edu/sim-data/) via the Mac OS X Command Line. After downloaded, I will move it to the Downloads folder in order to showcase some more options of `make_simlibs`.\n",
+    "In this example, we will download the `baseline_v1.4_10yrs.db` file from [here](https://lsst-web.ncsa.illinois.edu/sim-data/) via the use of the nix Command Line `curl` tool. After downloaded, I will move it to the Downloads folder in order to showcase some more options of `make_simlibs`.\n",
     "\n",
     "```shell\n",
     "curl -O https://lsst-web.ncsa.illinois.edu/sim-data/sims_featureScheduler_runs1.4/baseline/baseline_v1.4_10yrs.db\n",
     "\n",
-    "mv baseline_v1.4_10yrs.db Downloads/\n",
+    "mv baseline_v1.4_10yrs.db ${HOME}/Downloads/\n",
+    "\n",
+    "# The database file is now in ${HOME}/Downloads/baseline_v1.4_10yrs.db\n",
     "```\n",
     "\n",
     "### 2. Set up options<a name=\"chooseOptions\"></a>\n",
     "\n",
-    "- `data_root` $\\rightarrow$ path to the `OpSimSummary` package. In this case the package is in `/Users/user_name`.\n",
-    "- `dbname` $\\rightarrow$ relative path to the OpSim DataBases file. In this case it is `Downloads/baseline_v1.4_10yrs.db`.\n",
+    "- `data_root` $\\rightarrow$ path to the directory containing `dbname`. In this case the package is in `${HOME}`.\n",
+    "- `dbname` $\\rightarrow$ relative path to the OpSim DataBases file. In this case it is `Downloads/baseline_v1.4_10yrs.db`. (This could have also been done as `data_root`  $\\rightarrow$   `${HOME}/Downloads` and `dbname` $\\rightarrow$ `baseline_v1.4_10yrs.db` as used below.\n",
```

Comment:
It misses a parenthesis after `below.`.

Reply:
Should it be instead 
```
`dbname` $\\rightarrow$ relative path to the OpSim DataBases file from `data_root`.
```
?
In this way, I think the parenthesis make more sense.